extensions/aws/s3/provision/src/test/java/org/eclipse/dataspaceconnector/provision/aws/s3/S3StatusCheckerIntegrationTest.java

```diff
@@ -69,12 +67,23 @@ void isComplete_noResources_whenNotComplete() {
     }
 
     @Test
-    void isComplete_noResources_whenComplete() {
+    void isComplete_noResources_whenComplete() throws InterruptedException {
         //arrange
         putTestFile(PROCESS_ID + ".complete", getFileFromResourceName("hello.txt"), bucketName);
 
-        //act-assert
-        await().atMost(10, TimeUnit.SECONDS).until(() -> checker.isComplete(createTransferProcess(bucketName), emptyList()));
+        var tp = createTransferProcess(bucketName);
```

Comment:
We could extract a method that does this (that could have an API similar to the awaitility one), would improve test readability

Reply:
agree